### PR TITLE
BGDIDIC-154: Rename attributes ch.swisstopo.amtliches-gebaeudeadressverzeichnis

### PR DESCRIPTION
**Pull request by b4l**
https://mf-chsdi3.dev.bgdi.ch/feature_BGDIDIC-154/shorten/9699369a39

### CODE REVIEW

**Review by ltclm — APPROVED** [review]
thank you!

**Comment by b4l** [comment]
@ltclm in order to facilitate serving `adr_egaid` in the identify rest api we probably need to load this attribute into `bgdi_id` and set the id to bgdi_id in the model.

**Comment by ltclm** [comment]
no please dont use bgdi_id for that, it is an internal counter with no relation to the data. please create a new primary key column and use this column in the chsdi  model and sphinx query.

File: chsdi/models/vector/stopo.py [hunk 1]
```diff
@@ -3097,20 +3097,20 @@ class AmtlichesAdressVerzeichnis(Base, Vector):
     __table_args__ = ({'schema': 'vd', 'autoload': False})
     __template__ = 'templates/htmlpopup/addressverzeichnis.mako'
     __bodId__ = 'ch.swisstopo.amtliches-gebaeudeadressverzeichnis'
-    __label__ = 'id'
-    __queryable_attributes__ = ['id', 'bdg_egid', 'adr_edid', 'str_esid', 'str_label', 'adr_number']
-    id = Column('adr_egaid', BigInteger, primary_key=True)
+    __label__ = 'adr_egaid'
+    __queryable_attributes__ = ['adr_egaid', 'bdg_egid', 'adr_edid', 'str_esid', 'stn_label', 'adr_number']
+    id = Column('id', BigInteger, primary_key=True)
```
Comment (ltclm) [inline chsdi/models/vector/stopo.py:3102]: is it not possible to just referene adr_egaid as source here? Why do we have to fill two columns with the same value?

Comment (b4l) [reply chsdi/models/vector/stopo.py:3102]: @ltclm according to the ticket we want to return `adr_egaid` as attribute name. if my test were right it is not possible to reference the same column to different attributes in the model, hence the duplication.

Comment (ltclm) [reply chsdi/models/vector/stopo.py:3102]: ok, thank you!

**Pull request merged**